terraform/staging/main.tf

```diff
@@ -87,6 +87,18 @@ module "application" {
   sidekiq_task_desired_count = 2
   sidekiq_task_memory        = 8192
 
+  out_of_hours_scale_down = {
+    enabled = true
+    timings = {
+      workday_start = "0 7"
+      workday_end   = "30 19"
+    }

```

Comment:
I've looked at the load balancer graphs for number of requests to find sensible times here (erring on the side of caution).

Note in this case we're only scaling down the number of app instances to 2 rather than 4 - the application will still function, just aiming to handle fewer requests.

(We should consider autoscaling at some point, but that's not in scope right now)